elasticdl/python/data/parallel_transform.py

```diff
@@ -0,0 +1,81 @@
+import itertools
+from multiprocessing import Process, Queue
+
+
+def worker_loop(records, index_q, result_q, transform_fn):
```

Comment:
Add a docstring to explain those arguments.

Reply:
Done